script.module.dropbox/changelog.txt

```diff
@@ -1,6 +1,8 @@
 [B]Version 8.4.0[/B]
```

Comment:
Kodi Krypton uses the `<news>` addon.xml parameter as a changelog. Changelog.txt is no longer supported.